setuptools_scm/version.py

```diff
@@ -145,6 +148,31 @@ def guess_next_dev_version(version):
         return guess_next_version(version.tag, version.distance)
 
 
+def guess_next_simple_semver(version, distance, retain, increment=True):
+    parts = str(version).split('.', retain)
```

Comment:
These two lines use retain twice for the same meaning. Consider replacing with
```
parts = str(version).split('.', retain)[:-1]
```

Reply:
@jaraco this needs a different expression perhaps, bur the `[:-1]` is wrong, as t fails for versions where the parts contain less than retain elements